src/test/kotlin/sirius/db/mongo/MangoNightlyTest.kt

```diff
@@ -9,47 +9,56 @@
 package sirius.db.mongo
 
 import org.junit.jupiter.api.Tag
-import sirius.kernel.BaseSpecification
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import sirius.kernel.SiriusExtension
 import sirius.kernel.Tags
 import sirius.kernel.di.std.Part
 import sirius.kernel.health.HandledException
+import java.util.*
+import kotlin.test.assertEquals
 
 @Tag(Tags.NIGHTLY)
-class MangoNightlySpec extends BaseSpecification {
-
-    @Part
-    private static Mango mango
-
-            def "selecting over 1000 entities in queryList throws an exception"() {
-        given:
-        mango.select(MangoListTestEntity.class).delete()
-                and:
-                for (int i = 0; i < 1001; i++) {
-        def entityToCreate = new MangoListTestEntity()
-        entityToCreate.setCounter(i)
-        mango.update(entityToCreate)
-    }
-        when:
-        mango.select(MangoListTestEntity.class).queryList()
-                then:
-                thrown(HandledException)
+@ExtendWith(SiriusExtension::class)
+class MangoNightlyTest {
+    companion object {
+        @Part
+        private lateinit var mango: Mango
```

Comment:
For the next PRs: GitHub seems to not handle the diff very well when the companion object is placed at the top. Please place it at the bottom of the file